src/components/ContactUsHeaderSection/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const ContactUsHeaderSection = () => {
+  const { t } = useTranslation();
+  return (
+    <>
```

Comment:
do you think you could wrap the element with a div? just a suggestion

Reply:
Thanks for your review Ibrahim. After your comment I tried adding a wrapper div and made some styling as well. PTAL